docs/4.4/api-reference.md

```diff
@@ -796,14 +798,111 @@ if err := client.DeleteAccessRequest(ctx, accessReqID); err != nil {
 
 ## Certificate Authority
 
-It might be useful to retrieve your [Certificate Authority](architecture/authentication.md#ssh-certificates) through the API if it is rotating frequently.
+Teleport uses SSH Certificates to securely connect servers. To achieve this, the Auth server of a Teleport cluster acts as the [Certificate Authority](architecture/authentication.md#ssh-certificates) (CA), which signs SSH certificates for users and hosts in the cluster. The auth server uses separate CAs for users and hosts.
+
+You may want to use this API to manage your CA if:
+
+- You don't want to automate rotating certificates with `tctl` ([rotating CA with tctl](admin-guide.md#certificate-rotation))
```

Comment:
this sounds like it's giving up something better (esp. with `automating` in context). Maybe something like:
```
- You cannot use tctl to rotate certificates ([rotating CA with tctl](admin-guide.md#certificate-rotation)) for some reason
```